### ISSUE TITLE
11.0.2 bug in chinese Wrath client

### ISSUE THREAD

**Issue by Yemenghuan** [DeadlyBossMods/DeadlyBossMods#1178]
2x DBM-Core/modules/AnnoyingPopup.lua:251: attempt to call field 'DoesAddOnExist' (a nil value)
[string "@DBM-Core/modules/AnnoyingPopup.lua"]:251: in function `AnnoyingPopupCheckZone'
[string "@DBM-Core/DBM-Core.lua"]:3808: in function `CheckAvailableMods'
[string "@DBM-Core/DBM-Core.lua"]:3948: in function `func'
[string "@DBM-Core/modules/Scheduler.lua"]:163: in function <DBM-Core/modules/Scheduler.lua:153>

Can you fix it，Thanks！

**Comment by MysticalOS** [comment]
Right, it doesn't inherit the polyfill from core. I'll Get a fix out for this soon

**Comment by Yemenghuan** [comment]
> Right, it doesn't inherit the polyfill from core. I'll Get a fix out for this soon

Thank you again！

**Comment by Yemenghuan** [comment]
for “Add support for test anonymization”
1x DBM-Core/DBM-RangeCheck.lua:15: Cannot find a library instance of "LibDropDownMenu".
[string "=[C]"]: in function `error'
[string "@!!!Libs/LibStub/LibStub.lua"]:38: in function `GetLibrary'
[string "@DBM-Core/DBM-RangeCheck.lua"]:15: in main chunk

Locals:
(*temporary) = "Cannot find a library instance of "LibDropDownMenu"."


1x DBM-Core/DBM-InfoFrame.lua:16: Cannot find a library instance of "LibDropDownMenu".
[string "=[C]"]: in function `error'
[string "@!!!Libs/LibStub/LibStub.lua"]:38: in function `GetLibrary'
[string "@DBM-Core/DBM-InfoFrame.lua"]:16: in main chunk

Locals:
(*temporary) = "Cannot find a library instance of "LibDropDownMenu"."

**Comment by Yemenghuan** [comment]
1x DBM-Core/DBM-Core.lua:3979: attempt to call method 'IsRadarShown' (a nil value)
[string "@DBM-Core/DBM-Core.lua"]:3979: in function `handler'
[string "@DBM-Core/DBM-Core.lua"]:1046: in function <DBM-Core/DBM-Core.lua:1033>

1x DBM-Core/modules/AnnoyingPopup.lua:251: attempt to call field 'DoesAddOnExist' (a nil value)
[string "@DBM-Core/modules/AnnoyingPopup.lua"]:251: in function `AnnoyingPopupCheckZone'
[string "@DBM-Core/DBM-Core.lua"]:3808: in function `CheckAvailableMods'
[string "@DBM-Core/DBM-Core.lua"]:3948: in function `func'
[string "@DBM-Core/modules/Scheduler.lua"]:163: in function <DBM-Core/modules/Scheduler.lua:153>

**Comment by MysticalOS** [comment]
The last two errors seem to be caused by LibDropDownMenu dropping support for wrath and using only code compatible with cata, era, and mainline.
I'll poke that author about it maybe but it's tough cause none of us really have a wrath client to test. 

That commit you mention "add support for test anonymization" didn't actually do anything except pull a newer version of library is what I suspect.

**Comment by MysticalOS** [comment]
I fixed the first bug at top though. that's going out in alpha in moments

**Comment by Yemenghuan** [comment]
The 'DoesAddOnExist' is done，but need to change something
DBM-Core\modules\AnnoyingPopup.lua，line 168-line 175，"Wrath" Need to be changed to "WoTLK"
![1](https://github.com/user-attachments/assets/3cfdd567-55f3-4cb2-94a9-1db5e7b577df)

**Comment by MysticalOS** [comment]
ah didn't catch that
